interop-tests/src/test/scala/akka/grpc/scaladsl/PowerApiSpec.scala

```diff
@@ -52,7 +52,7 @@ abstract class PowerApiSpec(backend: String)
     with BeforeAndAfter
     with BeforeAndAfterAll {
 
-  override implicit val patienceConfig = PatienceConfig(5.seconds, Span(10, org.scalatest.time.Millis))
+  override implicit val patienceConfig: PatienceConfig = PatienceConfig(5.seconds, Span(10, org.scalatest.time.Millis))
```

Comment:
I thought we already needed that for Scala 3?

Reply:
I think it is because not all modules are compiled with Scala 3 (this but also a bit surprising if the interop tests aren't)